src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesSettings/ExternalDataSourcesSettings.test.js

```diff
@@ -0,0 +1,194 @@
+    await Pane('External data sources').has({ subtitle: '2 external data sources' });
+  });
+
+  test('renders the expcted number of MCL columns', async () => {
+    await MultiColumnList({ columnCount: 8 }).exists();
+  });
+
+  test('renders the expcted number of MCL rows', async () => {
+    await MultiColumnList({ rowCount: 2 }).exists();
+  });
+
+  test('renders expected columns', async () => {
+    await MultiColumnList({
+      columns: [
+        'Name',
+        'Type',
+        'URI',
+        'Is active',
+        'Trusted for title instance metadata',
+        'Sync status',
+        'Cursor',
+        'Last checked'
+      ],
+    }).exists();
+  });
+
+  test('renders expected name in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 0 }).has({ content: 'GOKb_TEST' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 0 }).has({ content: 'MR TEST' })
+    ]);
+  });
+
+  test('renders expected type in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 1 }).has({ content: 'org.olf.kb.adapters.GOKbOAIAdapter' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 1 }).has({ content: 'org.olf.kb.adapters.GOKbOAIAdapter' })
+    ]);
+  });
+
+  test('renders expected uri in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 2 }).has({ content: 'https://gokbt.gbv.de/gokb/oai/index' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 2 }).has({ content: 'https://mrtest.co.uk' })
+    ]);
+  });
+
+  test('renders expected is active in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 3 }).has({ content: 'Yes' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 3 }).has({ content: 'Yes' })
+    ]);
+  });
+
+  test('renders expected Trusted for title instance metadata in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 4 }).has({ content: 'No' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 4 }).has({ content: 'Yes' })
+    ]);
+  });
+
+  test('renders expected sync status in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 5 }).has({ content: 'idle' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 5 }).has({ content: '' })
+    ]);
+  });
+
+  test('renders expected cursor in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 6 }).has({ content: '2023-12-07T17:18:02Z' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 6 }).has({ content: '' })
+    ]);
+  });
+
+  test('renders expected last checked in the row', async () => {
+    Promise.all([
+      await MultiColumnListCell({ row: 0, columnIndex: 7 }).has({ content: '1701974599391' }),
+      await MultiColumnListCell({ row: 1, columnIndex: 7 }).has({ content: '' })
+    ]);
+  });
```

Comment:
I would like this test to also test the state change stuff...

ie clicking on an External data source should render a Pane with the right stuff in it.